Update the structure of Synthetic monitoring section (#1725)
elastic/observability-docs: docs/en/observability/monitor-uptime-synthetics.asciidoc

[[monitor-uptime-synthetics]]
= Uptime monitoring

++++
<titleabbrev>Uptime monitoring</titleabbrev>
++++

Synthetic monitoring extends traditional end-to-end testing techniques because it allows your tests to run continuously on the cloud.

> With synthetic monitoring in the {uptime-app}, you can assert that your application continues to work after a deployment by reusing

Author: Yea. I think this intro might need to be revisited to better introduce all of the Uptime app.

// geographic locations of each monitor.

// , explore
// endpoint status over time, drill down into specific monitors, and view a high-level
// snapshot of your environment at any point in time.

[discrete]
[[monitoring-synthetics]]
// [[synthetic-monitoring]]
=== Browser checks

// | *Browser monitor* | 

beta[] Real browser synthetic monitoring enables you to test critical actions and requests that an end-user would perform
on your site at predefined intervals and in a controlled environment.
The result is rich, consistent, and repeatable data that you can trend and alert on.

For example, you can test popular user journeys, like logging in, adding items to a cart, and checking
out -- actions that need to work for your users consistently.

Using the synthetics agent, you can run automated synthetic monitoring test suites on a real Chromium browser and
view each synthetic monitoring journey in the {uptime-app} side-by-side with your other monitors.

[role="screenshot"]
image::images/synthetic-app-overview.png[Synthetics app overview]

Alerting ensures that any degraded performance or broken actions are fixed before impacting your bottom line or customer

> experience.

Reviewer: Nit: to be precise here I'd say that "alerting _helps_ resolving", not that it resolves. I.e.

> Alerting helps you detect degraded performance or broken actions before your users do. By receiving alerts early, you can fix issues before they impact your bottom line or customer experience.

To get started writing user journeys, see <<synthetics-journeys>>.

// [TIP]
// ====
// Have a question? Want to leave feedback? Visit the
// https://discuss.elastic.co/tags/c/observability/uptime/75/synthetics[Synthetics discussion forum].
// ====

// [role="screenshot"]
// image::images/synthetic-app-overview.png[Synthetics app overview]

// beta[] The {uptime-app} uses {agent} to run automated synthetic monitoring test suites using a real Chromium browser.
// Using the `elastic-agent-complete` Docker container, create a *browser* monitor to mimic the user experience
// across multistep journeys and analyze what the browser displays for each step alongside a test script. 

// To get started, try out our {agent} using the `elastic-agent-complete` Docker container image. For details, refer
// to <<uptime-set-up,Get started using {agent} and {fleet}>>.

> // Or you might prefer to get started using our customizable Docker project template. This template provides two
> // types of sample tests: a two-step inline test and a packaged todo application with a custom suite of tests. For
> // more details, see <<uptime-set-up,Get started using Docker>>.

Reviewer: ```suggestion
// Alternatively, you can use our customizable Docker project template. This template provides two
// types of sample tests: a two-step inline test and a packaged todo application with a custom suite of tests. For
// more details, see <<uptime-set-up,Get started using Docker>>.
```

[discrete]
[[view-certificate-status]]
== TLS Certificates

The TLS Certificates page in the {uptime-app} lists the TLS certificates that are being monitored and 
shows the TLS certificate data in your indices.

In addition to the common name, associated monitors, issuer information, and SHA fingerprints,
an assigned status is derived from the threshold values in the <<configure-uptime-settings,Settings>> page.

[role="screenshot"]
image::images/tls-certificates.png[TLS certificates]

The table entries can be sorted by _status_ and _valid until_. You can use the search bar at the
top of the view to find values in most of the TLS-related fields in your Uptime indices.

> Additionally, you can select the *Alerts and rules* dropdown at the top of the page, and create a <<tls-certificate-alert,TLS rule>>.

Reviewer: ```suggestion
Additionally, you can select the *Alerts and rules* dropdown at the top of the page, and create a <<tls-certificate-alert,TLS rule>> to receive an alert when your certificate is about to expire.
```

[[monitor-uptime-synthetics]]
= Uptime and synthetic monitoring

++++
<titleabbrev>Uptime and synthetic monitoring</titleabbrev>
++++

[[uptime-monitors]]

The {uptime-app} uses {agent} to check the status of your services and applications periodically.
By adding the Elastic Synthetics integration to the policy used by your agent, you can monitor the
availability of network endpoints and services using the following Uptime monitors:

* <<monitoring-uptime>>
* <<monitoring-synthetics>>

[discrete]
[[monitoring-uptime]]
== Lightweight HTTP/S, TCP, and ICMP monitors

In the {uptime-app}, you can monitor the status of network endpoints using the following lightweight checks:

|===

> | *ICMP monitor* | Check the availability of your hosts. The ICMP monitor uses ICMP (v4 and v6) Echo

Reviewer 2: Can we flip this order? HTTP is the most common check, followed by perhaps a tie for TCP and ICMP, however I think it makes sense for TCP to be after HTTP because that puts the protocols in descending order of complexity.

status code and display the correct text.

| *ICMP monitor* | Check the availability of your hosts. The ICMP monitor uses ICMP (v4 and v6) Echo
Requests to check the network reachability of the hosts you are pinging. This will tell you whether the
host is available and connected to the network, but doesn't tell you if a service on the host is running or
not.

| *TCP monitor* | Monitor the services running on your hosts. The TCP monitor checks individual ports
to make sure the service is accessible and running.

|===

[role="screenshot"]
image::images/uptime-app.png[{uptime-app} in {kib}]

To set up your first monitor, see <<uptime-set-up>>.

// Along with getting notified when your <<tls-certificate-alert,TLS certificates>> are set to expire, you can also
// create an <<duration-anomaly-alert,Uptime duration anomaly >> rule to receive notifications based on the response durations for all of the

> // geographic locations of each monitor.

Reviewer 3: leftovers?

[discrete]
[[monitoring-synthetics]]
== Browser monitors

beta[] Real browser synthetic monitoring enables you to test critical actions and requests that an end-user would perform
on your site at predefined intervals and in a controlled environment.

> The result is rich, consistent, and repeatable data that you can trend and alert on.

Reviewer: I love this paragraph! Informative, concise, and impactful.

For example, you can test popular user journeys, like logging in, adding items to a cart, and checking
out -- actions that need to work for your users consistently.

Using the synthetics agent, you can run automated synthetic monitoring test suites on a real Chromium browser and

> view each synthetic monitoring journey in the {uptime-app} side-by-side with your other monitors.

Reviewer: This might be a little confusing to users because what actually gets journeys to the Uptime app is Heartbeat (or the Elastic Agent, which bundles Heartbeat).

I'd probably say that the same journeys people write and run directly with the agent could also be run using Heartbeat, so that people see results in the uptime app (at the end of the day Heartbeat also uses the agent, but that's unnecessary info, so we don't need to say that here)

Author: I made an attempt, but I'm not sure if it addresses your comment.

[role="screenshot"]
image::images/synthetic-app-overview.png[Synthetics app overview]

Alerting helps you detect degraded performance or broken actions before your users do.
By receiving alerts early, you can fix issues before they impact your bottom line or customer experience.

To get started with browser monitors, <<uptime-set-up, get set up>> and then <<synthetics-journeys, write a synthetic journey>>.

// [TIP]
// ====
// Have a question? Want to leave feedback? Visit the
// https://discuss.elastic.co/tags/c/observability/uptime/75/synthetics[Synthetics discussion forum].

> // ====

Reviewer 3: leftovers?
